### ISSUE TITLE
[a11y] CRT pagination arrows missing label

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#284]
The pagination of the view all table arrows are not labeled for a screenreader
forward arrow doesn't have a label
I think the back arrow is labeled "go back to page 1"

Expected:
`>` label is "Next"
`<` label is  "Back"

**Comment by Jacklynn** [comment]
🗣➡🎉
